src/components/styles/GlobalStyles.tsx

```diff
@@ -17,6 +17,7 @@ const globalStyles = css`
   body {
     font-family: "Euclid Circular A", "Helvetica Neue", Helvetica, Arial,
       sans-serif;
+    font-size: 13px;
```

Comment:
The provider already defaults to 13 but it only only applies to LeafyGreen elements anyway—it's helpful to set an accurate base size so that non-LG Typography elements (of which there are admittedly few!) are at the right scale.